plugins/ScheduledReports/ScheduledReports.php

```diff
@@ -343,12 +344,39 @@ public function sendReport($reportType, $report, $contents, $filename, $prettyDa
             $this->markReportAsSent($report, $period);
         }
 
+        $subscriptionModel = new SubscriptionModel();
+        $subscriptionModel->updateReportSubscriptions($report['idreport'], $emails);
+        $subscriptions = $subscriptionModel->getReportSubscriptions($report['idreport']);
+
+        $tokens = [];
+
+        foreach ($subscriptions as $subscription) {
+            $tokens[$subscription['email']] = $subscription['token'];
+        }
```

Comment:
Think this could be shortened to `$tokens = array_column($subscriptions, 'token', 'email');`

Reply:
Not really important, just fyi